### PR TITLE
feat(crumbs-v2): Highlight text search results

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Description:**
. Rename the main file breadcrumbs.tsx to index.tsx
. Pass down the prop searchTerm 
. Use the component Hightlight together with the searchTerm value
. Fix back slash string comparison issue 

Next Steps:

create crumb's React.context provider and consumer, so it won't have to pass the down the props + 
Improve performance 

closes: https://app.asana.com/0/1158284503473033/1180990285597801

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/index.tsx [hunk 1]
```diff
@@ -233,28 +233,42 @@ class Breadcrumbs extends React.Component<Props, State> {
...
-              .toLocaleLowerCase()
-              .includes(searchTerm);
-          })
-        );
+    if (!searchTerm.trim()) {
+      return breadcrumbs;
+    }
+
+    // Slightly hacky, but it works
+    // the string is being stringfy here in order to match exactly the same stringfy string of the loop
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/index.tsx:249]:
```suggestion
    // the string is being `stringfy`d here in order to match exactly the same `stringfy`d string of the loop
```

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/index.tsx:249]: nice...thanks :)

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/level.tsx [hunk 2]
```diff
@@ -1,28 +1,50 @@
...
+      return (
+        <StyledTag color="orange400">
+          <Highlight text={searchTerm}>{t('warning')}</Highlight>
+        </StyledTag>
+      );
     default:
-      return <Tag>{level || t('undefined')}</Tag>;
+      return (
+        <Tag>
+          <Highlight text={searchTerm}>{t('undefined')}</Highlight>
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/level.tsx:45]: Do you need the `level || t('undefined')` here still?

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/level.tsx:45]: good catch!!

Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/level.tsx:45]:
```suggestion
          <Highlight text={searchTerm}>{level || t('undefined')}</Highlight>
```

**Review by markstory — APPROVED** [review]

**Pull request merged**